### ISSUE TITLE
Question about pytorch

### ISSUE THREAD

**Issue by LJ-Hao** [davabase/whisper_real_time#52]
`parameter logits (Tensor of shape (1, 51864)) of distribution Categorical(logits: torch.Size([1, 51864])) to satisfy the constraint IndependentConstraint(Real(), 1), but found invalid values: tensor([[nan, nan, nan,  ..., nan, nan, nan]], device='cuda:0')`
I have tried [this] (https://github.com/openai/whisper/discussions/1068) but it did not work.
I got `!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! 1.5mm 1.5mm 1.5mm 1.5mm 1.5mm 1.5mm 1.5mm 1.5mm 1.5mm 1.5mm 1.5mm 1.5mm 1.5mm 1.5mm`

**Comment by LJ-Hao** [comment]
I have solved this problem on Jetson.`nvidia@nvidia-desktop:~$ arecord -D hw:2,0 --dump-hw-params` will set my mic to 16000hz. So problem solved.
